src/main/java/in/org/projecteka/hiu/user/Session.java

```diff
@@ -5,4 +5,5 @@
 @Value
 public class Session {
     String accessToken;
+    boolean isVerified;
```

Comment:
I'm just wondering why do we need this, in your UI you can decode the token and use that value.

Reply:
But that would need the sharedSecret to decode right?

Reply:
Ohk Got it. shared secret is not required. We can decode it in frontend. But didn't we decide to change the API contract to be this.